docs/contributors/continuous-integration.md

```diff
@@ -10,10 +10,8 @@ https://ci.infra.status.im/job/nim-waku/
 
 It currently consists of four jobs:
 
-* [manual](https://ci.infra.status.im/job/nim-waku/job/manual/) - For manually executing builds using parameters.
```

Comment:
I have the impression that the `manual` job is still valid.

Reply:
Good catch, thank you!